graphql/typeDefs.ts

```diff
@@ -206,4 +214,14 @@ export default gql`
     student: User!
     lesson: Lesson!
   }
+
+  type Module {
+    id: Int!
+    author: User!
+    authorId: Int!
+    lesson: Lesson!
+    lessonId: Int!
```

Comment:
`lessonId` is inside the lesson object, so this is redundant here. Users could just use `lesson.id`